database/migration/V20160614_1845__createUsersTable.sql

```diff
@@ -7,4 +7,5 @@ CREATE TABLE `users` (
   `created` DATETIME NULL DEFAULT CURRENT_TIMESTAMP,
   `updated` DATETIME NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   PRIMARY KEY (`id`),
-  UNIQUE INDEX `email_UNIQUE` (`email` ASC));
```

Comment:
Put this migration back the way it was. The migration chain has been corrupted by splitting this line. If you're not running the dev migrations before you submit a PR, you're doing it wrong.